server/db/models/index.js

```diff
@@ -8,6 +9,8 @@ const Product = require('./product')
  *    BlogPost.belongsTo(User)
  */
 
+Order.belongsTo(User)
```

Comment:
Obviously based on discussion this morning I can approve these anticipating that we're restructuring